driver-core/src/main/java/com/datastax/driver/core/policies/ErrorAwarePolicy.java

```diff
@@ -201,12 +199,13 @@ public Builder withInclusionThreshold(double errorsPerSecThreshold) {
          * <p/>
          * The default implementation will exclude from the error counting, the following exception types :
          * <ul>
-         *     <li>{@link ReadTimeoutException}</li>
-         *     <li>{@link WriteTimeoutException}</li>
-         *     <li>{@link UnavailableException}</li>
-         *     <li>{@link ReadFailureException}</li>
-         *     <li>{@link WriteFailureException}</li>
+         * <li>{@link ReadTimeoutException}</li>
```

Comment:
Does your auto formatter pull away the indentation like mine does?  I thought it was weird when my IDE did that, but was wondering how it behaves for you all.